tutorlimesurvey/plugin.py

```diff
@@ -20,15 +20,16 @@
         # Each new setting is a pair: (setting_name, default_value).
         # Prefix your setting names with 'LIMESURVEY_'.
         ("LIMESURVEY_VERSION", __version__),
-        ("LIMESURVEY_DOCKER_IMAGE", "docker.io/acspri/limesurvey:6.1.0"),
+        ("LIMESURVEY_DOCKER_IMAGE", "docker.io/edunext/limesurvey:15.0.0"),
+        ("LIMESURVEY_DOCKER_BASE_IMAGE", "docker.io/acspri/limesurvey:6.1.0"),
```

Comment:
Should we start using [this image](https://hub.docker.com/r/adamzammit/limesurvey)?

Reply:
You made reference to [this repository](https://github.com/adamzammit/limesurvey-docker) in the description of the PR.

Reply:
This variable is not used anywhere, does it have any pupose?

Reply:
I suppose that is what we use to build our own, right?

Also, the image itself makes no difference. It is published by the same person in both places.

but we should move to 6.2.0 which is now available. https://hub.docker.com/layers/acspri/limesurvey/6.2.0/images/sha256-7020d0a3e849726efe400ace3d27e2fc270d77bb7cb914b8f2e8b97b8edf44b6?context=explore

Reply:
Yes, we built the production image on top of it. Thanks!